src/utils/sortElements.ts

```diff
@@ -0,0 +1,16 @@
+function sortElements(
+  elements: string[],
+  criterion: "A-Z" | "Z-A" | "none"
```

Comment:
please abstract the `sort` type on src/types/settings.d.ts and reuse it here. 

also mark it as optional so it will not give you conflicts with settings.d.ts